rest/rest-resources/src/main/java/org/apache/brooklyn/rest/transform/EffectorTransformer.java

```diff
@@ -83,7 +84,8 @@ public static EffectorSummary effectorSummaryForCatalog(Effector<?> effector) {
                 .context(entity).timeout(ValueResolver.REAL_QUICK_WAIT).getMaybe();
             return new ParameterSummary(parameterType.getName(), parameterType.getParameterClassName(), 
                 parameterType.getDescription(), 
-                WebResourceUtils.getValueForDisplay(defaultValue.orNull(), true, false));
+                WebResourceUtils.getValueForDisplay(defaultValue.orNull(), true, false),
+                Sanitizer.IS_SECRET_PREDICATE.apply(parameterType.getName()));
```

Comment:
It looks like you added `parameterType.isSensitive`, but then don't use it here (and instead rely on `Sanitizer`). As I said in earlier comment, I think that's fine - in fact probably best to delete `isSensitive` if we're not going to use it everywhere.